### PR TITLE
exa: use create_wrksrc

### PR DESCRIPTION
**Pull request by tranzystorek-io**
#### Testing the changes
- I tested the changes in this PR: **briefly**

### CODE REVIEW

File: srcpkgs/exa/template [hunk 1]
```diff
@@ -15,9 +17,9 @@ checksum="ff0fa0bfc4edef8bdbbb3cabe6fdbd5481a71abbbcc2159f402dea515353ae7c
  531596a1ef2a757c7728087529528150e6eb52bb8224fe575aa00a5f1b762849"
 
 post_extract() {
-	mkdir -p accoutrements
-	mv ../man accoutrements/
-	mv ../completions accoutrements/
+	mkdir -p "${build_wrksrc}/accoutrements"
```
Comment (paper42) [inline srcpkgs/exa/template:20]:
```suggestion
	mkdir "${build_wrksrc}/accoutrements"
```
post_extract is guaranteed to only run once, so -p is not required

Comment (tranzystorek-io) [reply srcpkgs/exa/template]: Done

**Comment by tranzystorek-io** [comment]
Please hold off on merging, I realised the second distfile delivers manpages and completions that are already present in the source tarball 😆

**Comment by tranzystorek-io** [comment]
Should be good to merge now 👍

File: srcpkgs/exa/template [hunk 2]
```diff
@@ -3,28 +3,33 @@ pkgname=exa
...
-post_extract() {
-	mkdir -p accoutrements
-	mv ../man accoutrements/
-	mv ../completions accoutrements/
-}
+build_options="man"
+desc_option_man="Use pandoc for manpages"
+
+case "$XBPS_MACHINE" in
+	x86_64*|i686|ppc64le*|ppc64) build_options_default="man" ;;
```
Comment (paper42) [inline srcpkgs/exa/template:19]: because of this restriction, aren't pre-generated manpages better?

Comment (tranzystorek-io) [reply srcpkgs/exa/template:19]: I can take it back, but is that such a big deal?

Comment (paper42) [reply srcpkgs/exa/template:19]: All cross compiled architectures don't have pandoc actually, this condition isn't exactly correct.

Comment (tranzystorek-io) [reply srcpkgs/exa/template]: Reverted to previous solution, sorry for the commotion

**Pull request merged**